### PR TITLE
Fix filter

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ export function getFilters() {
  * @param name
  * @returns {Promise<Response>}
  */
-export function deleteFilterByName(name) {
+export function deleteFilterById(name) {
```
Comment (FranckLecuyer) [inline src/utils/rest-api.js]: Maybe change the parameter 'name' into 'id' ?

File: src/utils/rest-api.js [hunk 2]
```diff
@@ -295,7 +295,7 @@ export function deleteFilterByName(name) {
  * Get filter by name
  * @returns {Promise<Response>}
  */
-export function getFilterByName(name) {
+export function getFilterById(name) {
```
Comment (FranckLecuyer) [inline src/utils/rest-api.js]: Same as above

File: src/utils/rest-api.js [hunk 3]
```diff
@@ -306,7 +306,7 @@ export function getFilterByName(name) {
  * @param newName
  * @returns {Promise<Response>}
  */
-export function renameFilterByName(oldName, newName) {
+export function renameFilter(oldName, newName) {
```
Comment (FranckLecuyer) [inline src/utils/rest-api.js]: Maybe rename parameter 'oldName' to 'oldId' ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/filter-list.js [hunk 4]
```diff
@@ -201,13 +206,12 @@ const FilterList = () => {
...
 
     const newFilter = (name, type) => {
-        setCurrentItemName(name);
         currentEdit.current = {
             name: name,
             type: type === ScriptTypes.SCRIPT ? type : 'LINE',
             transient: true,
         };
-        save();
+        save(name);
```
Comment (niconoir) [inline src/components/filter-list.js]: Parameter name not needed

**Review by niconoir — CHANGES_REQUESTED** [review]

File: src/utils/rest-api.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/utils/rest-api.js]: Maybe change also function comment 'Delete filter by name' ?

File: src/utils/rest-api.js (hunk 2 above)
Comment (FranckLecuyer) [reply src/utils/rest-api.js]: Maybe change also function comment 'Get filter by name' ?

File: src/utils/rest-api.js (hunk 3 above)
Comment (FranckLecuyer) [reply src/utils/rest-api.js]: Maybe change also function description 'Rename filter by name' ?

**Review by FranckLecuyer — COMMENTED** [review]
See my new remarks on functions descriptions in rest-api.js

**Review by niconoir — APPROVED** [review]

File: src/utils/rest-api.js (hunk 3 above)
Comment (JacquesBorsenberger) [reply src/utils/rest-api.js]: done

File: src/utils/rest-api.js (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/utils/rest-api.js]: done

File: src/utils/rest-api.js (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/utils/rest-api.js]: done

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**